Source/LuaBridge/detail/Namespace.h

```diff
@@ -1041,6 +1041,12 @@ class Namespace : public detail::Registrar
             lua_newtable(L); // Stack: pns, ns, propset table (ps)
             lua_rawsetp(L, -2, detail::getPropsetKey()); // ns [propsetKey] = ps. Stack: pns, ns
 
+			if (Security::hideMetatables())
+			{
+				lua_pushboolean(L, 0);
+				rawsetfield(L, -2, "__metatable");
+			}
+
```

Comment:
Spaces should be used instead of tabs.

Reply:
@Mellnik ?

Reply:
Is it okay for you when you just push the change under your name?